cassandra-dac/src/main/java/org/sunbird/helper/CassandraConnectionManagerImpl.java

```diff
@@ -0,0 +1,243 @@
+package org.sunbird.helper;
+
+import com.datastax.driver.core.AtomicMonotonicTimestampGenerator;
+import com.datastax.driver.core.AuthProvider;
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Host;
+import com.datastax.driver.core.HostDistance;
+import com.datastax.driver.core.Metadata;
+import com.datastax.driver.core.PoolingOptions;
+import com.datastax.driver.core.ProtocolVersion;
+import com.datastax.driver.core.QueryLogger;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.policies.DefaultRetryPolicy;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.thrift.transport.TTransportException;
+import org.cassandraunit.CQLDataLoader;
+import org.cassandraunit.dataset.cql.ClassPathCQLDataSet;
+import org.cassandraunit.utils.EmbeddedCassandraServerHelper;
+import org.sunbird.common.Constants;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.models.util.PropertiesCache;
+import org.sunbird.common.responsecode.ResponseCode;
+
+/**
+ * Created by arvind on 11/10/17.
+ */
+public class CassandraConnectionManagerImpl implements CassandraConnectionManager {
+
+  private String mode;
+  private static Map<String, Session> cassandraSessionMap = new HashMap<>();
+  private static Map<String, Cluster> cassandraclusterMap = new HashMap<>();
+
+  static{
+    registerShutDownHook();
+  }
+
+  public CassandraConnectionManagerImpl(String mode) {
+    this.mode = mode;
+  }
+
+  @Override
+  public boolean createConnection(String ip, String port, String userName, String password,
+      String keyspace) {
+    boolean connectionCreated = false;
+    if (mode.equalsIgnoreCase(JsonKey.EMBEDDED_MODE)) {
```

Comment:
If it is embedded then return embedded object other wise return standalone object always.